api/helpers/uploadFileToStorage.js

```diff
@@ -0,0 +1,61 @@
+const multerS3 = require('multer-s3');
+const multer = require('multer');
+const AWS = require('aws-sdk');
+const s3 = new AWS.S3({ region: 'eu-west-2' });
+
+const inDevEnvironment = process.env.NODE_ENV === 'development';
```

Comment:
Rather than all the checks to see if you are in dev mode, just use localstack to create an actual bucket to simulate uploads

Reply:
This is a good point. We had a discussion about this at CYB before this feature was implemented and we opted to upload files locally without localstack because it was easier. So Jon (don't know if you remember him) implemented local uploads before I picked the project up. 

I'll have a chat with the others and get their thoughts on this but in all honesty, this will have to be done post MVP unless you think it is essential?